### PR TITLE
Update Gradle wrapper to 2.14.1

### PR DESCRIPTION
**Pull request by jamierocks**
- Rename plugin to 'net.fabricmc.loom' from 'fabric-loom'

- Add .gitattributes

- Use the same .gitignore from fabric-base

- Add license headers

- Use -SNAPSHOT for builds

### CODE REVIEW

**Comment by shadowfacts** [comment]
If you're going to update Gradle, update everything to 3.0?

File: build.gradle [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+
+    id 'net.minecrell.licenser' version '0.2.1'
 }
 
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+group = 'net.fabric'
+archivesBaseName = project.name.toLowerCase()
+version = '0.0.1-SNAPSHOT'
```
Comment (modmuss50) [inline build.gradle]: I dont see the reason behind doing this. Can you explain this?

Comment (shadowfacts) [inline build.gradle]: http://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it

**Comment by jamierocks** [comment]
3.0 is not released.

**Comment by shadowfacts** [comment]
https://gradle.org/gradle-download/

> The latest release of Gradle is version 3.0, released on 15th August 2016.

**Comment by modmuss50** [comment]
Have you tested it on 3?

**Comment by jamierocks** [comment]
I tested building which works.

**Comment by modmuss50** [comment]
We we are going to use gradle 3.0, all the projects should use it, so I will test this, with the base.

I am slightly wary of new things

**Pull request merged**